pkg/lightning/backend/importer.go

```diff
@@ -348,19 +348,22 @@ func checkVersion(component string, actual, requiredMinVersion, requiredMaxVersi
 	// actual version must be within [requiredMinVersion, requiredMaxVersion).
 	if actual.Compare(requiredMinVersion) < 0 {
 		return errors.Errorf(
-			"%s version too old, required to be in [%s, %s), found '%s'",
+			"%s version too old, major version required to be in [%d, %d), found '%s'",
 			component,
-			requiredMinVersion,
-			requiredMaxVersion,
+			requiredMinVersion.Major,
+			requiredMaxVersion.Major,
 			actual,
 		)
 	}
```

Comment:
the min version check should continue report the full version. note that Importer Backend's min requirement is 2.1, not 2.0.